configs/model/cora.yaml

```diff
@@ -1,6 +1,7 @@
 _target_: src.models.cora_module.CoraLitModule
 lr: 2e-5
-
+save_name: ${model_name}
+model_dir: ${paths.model_dir}
```

Comment:
What's the difference in functions between model_dir and model:cache_dir? I guess both used to store pre-trained models?